cypress/integration/registration.spec.js

```diff
@@ -0,0 +1,243 @@
+        cy.get('.nav-link').eq(2).click();
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/register');
+        cy.get('#first-name').type('Dina');
+        cy.get('#last-name').type('Bolta');
+        cy.get('#email').type('.dina729@gmail.com');
+        cy.get('#password').type('test1234');
+        cy.get('#password-confirmation').type('test1234');
+        cy.get('.form-check-input').click();
+        cy.get('button[type="submit"]').click();
+    });
+
+    it('Register without password and password confirmation', () => {
+        cy.visit('/');
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/');
+        cy.get('.nav-link').eq(2).click();
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/register');
+        cy.get('#first-name').type('Dina');
+        cy.get('#last-name').type('Bolta');
+        cy.get('#email').type('dina729@gmail.com');
+        cy.get('.form-check-input').click();
+        cy.get('button[type="submit"]').click();
+    });
+
+    it('Register without password confirmation', () => {
+        cy.visit('/');
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/');
+        cy.get('.nav-link').eq(2).click();
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/register');
+        cy.get('#first-name').type('Dina');
+        cy.get('#last-name').type('Bolta');
+        cy.get('#email').type('dina729@gmail.com');
+        cy.get('#password').type('test1234');
+        cy.get('.form-check-input').click();
+        cy.get('button[type="submit"]').click();
+    });
+
+    it('Register with invalid password-less than 8 characters', () => {
+        cy.visit('/');
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/');
+        cy.get('.nav-link').eq(2).click();
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/register');
+        cy.get('#first-name').type('Dina');
+        cy.get('#last-name').type('Bolta');
+        cy.get('#email').type('dina729@gmail.com');
+        cy.get('#password').type('test12');
+        cy.get('#password-confirmation').type('test12');
+        cy.get('.form-check-input').click();
+        cy.get('button[type="submit"]').click();
+    });
+
+    it('Register with invalid password-without a digit', () => {
+        cy.visit('/');
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/');
+        cy.get('.nav-link').eq(2).click();
+        cy.url().should('eq', 'https://gallery-app.vivifyideas.com/register');
+        cy.get('#first-name').type('Dina');
+        cy.get('#last-name').type('Bolta');
+        cy.get('#email').type('dina729@gmail.com');
+        cy.get('#password').type('testtestt');
+        cy.get('#password-confirmation').type('testtestt');
+        cy.get('.form-check-input').click();
+        cy.get('button[type="submit"]').click();
+    });
+
+    it('Register if password and password confirmation don\'t match', () => {
+        cy.visit('/');
```

Comment:
Nevezano za domaci - dobra upotreba backslash-a, ali u nazivu it-a izgleda ne moras da escape-ujes taj navodnik, cim je podigao i backslash gore.